src/app/components/form/DualCurrencyField/index.tsx

```diff
@@ -61,6 +61,14 @@ export default function DualCurrencyField({
     />
   );
 
+  // run effect on input mount to ignore wheel/scroll event
+  useEffect(() => {
+    const ignoreScroll = (evt: globalThis.WheelEvent) => {
+      evt.preventDefault();
+    };
+    inputEl.current && inputEl.current.addEventListener("wheel", ignoreScroll);
```

Comment:
I think, the event listener should be removed on unmount.

I would add:
```TS
  return () => inputEl.current && inputEl.current.addEventListener("wheel", ignoreScroll);
```

to the hook.

What do you think?

Reply:
@lisabaut At first I thought the same, but this event is on input ref and on unmount that will become null.

Reply:
Yes, I think you are right. Thanks for the pointer.

Reply:
@lisabaut There's a way to do that, we can assign ref to another variable, and then we can use that variable to remove the event listener.

```
const elem = inputEl.current;
return () => elem?.removeEventListener("wheel", ignoreScroll);
```
Should I do that?

Reply:
I am also still thinking about this ref and am reading about it again.

Yes, let´s rather remove the event listener even though it is highly likely that the ref is `null` on unmount. 👍 
Even though I guess this is only the case when using `useLayoutEffect` and not with `useEffect`

So like:

```TS
  useEffect(() => {
    const inputRef = inputEl.current;
    const ignoreScroll = (evt: globalThis.WheelEvent) => {
      evt.preventDefault();
    };

    inputRef && inputRef.addEventListener("wheel", ignoreScroll);

    return () => {
      inputRef && inputRef.removeEventListener("wheel", ignoreScroll);
    };
  }, [inputEl]);
```

Same code with storing the `ref.current` in a local var inside the `useEffect`:
https://bobbyhadz.com/blog/react-ref-add-event-listener

In depth reading again:
https://blog.thoughtspile.tech/2021/05/17/everything-about-react-refs/

Reply:
@lisabaut I have added removeEventListener on unmount